spec

```diff
@@ -2389,15 +2389,15 @@ endseg
 beginseg
     name "ovl_En_Go"
     include "build/src/overlays/actors/ovl_En_Go/z_en_go.o"
-    include "build/data/overlays/actors/z_en_go.data.o"
-    include "build/data/overlays/actors/z_en_go.rodata.o"
+    // include "build/data/overlays/actors/z_en_go.data.o"
+    // include "build/data/overlays/actors/z_en_go.rodata.o"
```

Comment:
if it OKs, the .data.o and .rodata.o files should be deleted, and the lines should be removed, not commented out. the same goes for ovl_En_Go2 just below this.